config/grafana/watcher.json

```diff
@@ -21,17 +21,254 @@
   "editable": true,
   "gnetId": null,
   "graphTooltip": 0,
-  "id": 41,
   "links": [],
   "panels": [
+    {
```

Comment:
Small suggestion:
- Make the image dashboard more tiny and put the unready deployments in the same row
- Just looks better to the eye, have a look at the screenshot with my suggestion

Before:

![Screenshot 2024-04-09 at 10 45 21](https://github.com/kyma-project/lifecycle-manager/assets/48282931/7db61087-4b09-446e-8618-00f750ac9373)

After:

![Screenshot 2024-04-09 at 10 45 05](https://github.com/kyma-project/lifecycle-manager/assets/48282931/84c93225-9b15-482c-92ed-4b7e772d3746)